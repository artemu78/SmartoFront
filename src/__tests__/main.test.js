import React from 'react';
import Button from '../components/common/button';
import TextInput from '../components/common/textinput';
import renderer from 'react-test-renderer';

test('<Button text=Send>', () => {
  const _button = renderer.create(<Button text='Send'/>);
  let json_string = _button.toJSON();
  expect(json_string).toMatchSnapshot();
});

test('<TextInput text=Send name=testname value=kot readOnly>', () => {
  const _button = renderer.create(
    <TextInput text='Send' name='testname' value='kot' readOnly>
      <div>hello world</div>
    </TextInput>
  );
  let json_string = _button.toJSON();
  expect(json_string).toMatchSnapshot();
});
