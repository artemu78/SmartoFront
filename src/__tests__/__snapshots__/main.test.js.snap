// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button text=Send> 1`] = `
<div>
  Send
</div>
`;

exports[`<TextInput text=Send name=testname value=kot readOnly> 1`] = `
<div>
  Send
   
  <div>
    hello world
  </div>
  <br />
  <input
    defaultValue="kot"
    name="testname"
    readOnly={true}
    type="text"
  />
</div>
`;
